1_fetch.R

```diff
@@ -0,0 +1,22 @@
+p1_targets_list <- list(
+
+# Get common parameter groups and WQP CharacteristicNames
+tar_target(
```

Comment:
I think the spacing here is off. All of these things within the `list()` should be indented one.